packages/core/botpress/src/web/components/Content/Select/index.js

```diff
@@ -115,7 +115,9 @@ class SelectContent extends Component {
   }
 
   resetCreateContent = (resetSearch = false) => response => {
-    const item = Object.assign(this.state.newItemData, { id: response.data })
+    const newItemData = this.state.newItemData || {}
+    const { data: id } = response
+    const item = Object.assign(newItemData, { id })
```

Comment:
@dmytropanontko , could it be oneliner like `const item = Object.assign(this.state.newItemData || {}, { id: response.data })`?

Reply:
I use `response.data` twice. I think it would be better leave as it is or no?